cmd/keytransparency-client/cmd/root.go

```diff
@@ -77,13 +79,18 @@ func init() {
 	cobra.OnInitialize(initConfig)
 	RootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.keytransparency.yaml)")
 
-	RootCmd.PersistentFlags().String("kt-url", "", "URL of Key Transparency server")
+	RootCmd.PersistentFlags().String("kt-url", "35.184.134.53:8080", "URL of Key Transparency server")
 	RootCmd.PersistentFlags().String("kt-cert", "genfiles/server.crt", "Path to public key for Key Transparency")
+	RootCmd.PersistentFlags().Bool("autoconfig", true, "Fetch config info from the server's /v1/domain/info")
+	RootCmd.PersistentFlags().Bool("insecure", false, "Skip TLS checks")
+
 	RootCmd.PersistentFlags().String("vrf", "genfiles/vrf-pubkey.pem", "path to vrf public key")
 
 	RootCmd.PersistentFlags().String("log-key", "genfiles/trillian-log.pem", "Path to public key PEM for Trillian Log server")
 	RootCmd.PersistentFlags().String("map-key", "genfiles/trillian-map.pem", "Path to public key PEM for Trillian Map server")
```

Comment:
Suggestion: Maybe add a comment stating that these flags will be set automatically if `--autoconfig=true`? Or, even add this to the viper description string. Otherwise, it might be unclear what will be set automatically and what not.

(If `--autoconfig=true`, which is default, it doesn't matter what you set for the `vrf`, `log-key`, and `map-key`)